docs/sources/write/shortcodes/index.md

```diff
@@ -907,6 +907,46 @@ You should generally avoid using this shortcode because every documentation page
 {{</* table-of-contents */>}}
 ```
 
+## Tabs
+
+The `tabs` shortcode creates generic tabbed content.
+The website saves the selected tab to browser storage and persists it across navigation.
+
+You create a tab using the `tab-content` shortcode within the `tabs` shortcode.
+The inner of the `tab-content` can be any Markdown.
+
+{{< admonition type="note" >}}
+You can't nest the `tabs` shortcode in a `tab-content` shortcode but you can use the `code` shortcode instead.
```

Comment:
Maybe add an example of this as a secondary example. I don't really know what the use case is here, so I'm not immediately grokking why this guidance is here.